.env.development

```diff
@@ -3,3 +3,4 @@ REACT_APP_MAPLIBRE_MAP_STYLE=https://api.maptiler.com/maps/8ecc366c-2d3e-45cf-b3
 REACT_APP_CLIENT_ID=2es98bqdkc6clcneghvbiu997u
 REACT_APP_USERPOOL_ID=eu-central-1_JRz1teeaq
 REACT_APP_REQUESTS_SOURCE=fakeRequestsV2
+REACT_APP_SHOW_REGIONS=0
```

Comment:
The argument I used before was: you can change the behavior of the frontend without a need to send a PR.
So say Daniel wants to disable/enable regions, instead of creating a github repo clone, finding the necessary file with the constant, changing it, sending a PR, waiting for approval, merging, he can just go the Github UI, change one value in the centralized place and click a button to rebuild.

Why is this not a convincing argument for you?

Reply:
It's not because it won't work. 

This application is a frontend application. process.env will work in build environment (Node is used to build the JS code). But to build the project, you have to trigger the build somehow. I think the easiest way to trigger the build is via a code-push. So you can change the variable in the code, make codepush, and the result will be the same. 

Unless, there is a way of easily rebuild the project using aws GUI, and say, Daniel, has acccess to it and knows how to use it. 

I don't :) 

If I had to change it, I'd prefer to have it in the code.

Reply:
But you can be right, and it can be easily rebuild with different variable, then I'll learn sth new, that's why I am not blocking it.

Reply:
Yeah, maybe you can trigger a GH action using GH gui...

Reply:
Yes, exactly, it's just 2 mouse clicks away from github's UI. 
My grandma can do it. But she cannot submit a code change.

Reply:
To even add a bit more. I already had to use this approach once, when the DEV frontend got broken while Gus was having a meeting, so I had to do a setting flip and publish real quick.

Reply:
Changed 0 to false, 1 to true.